homeassistant/components/hassio/manifest.json

```diff
@@ -6,6 +6,6 @@
   "documentation": "https://www.home-assistant.io/integrations/hassio",
   "iot_class": "local_polling",
   "quality_scale": "internal",
-  "requirements": ["aiohasupervisor==0.2.2b4"],
+  "requirements": ["aiohasupervisor==0.2.2b5"],
```

Comment:
Six files should have diff. See https://github.com/home-assistant/core/pull/133652